lizmap/modules/lizmap/lib/Project/Repository.php

```diff
@@ -152,7 +155,7 @@ public function getACAOHeaderValue($referer)
         }
 
         if (is_string($origins)) {
-            $origins = preg_split('/\\s*,\\s*/', $origins);
+            $origins = preg_split('/\s*,\s*/', $origins);
```

Comment:
I wonder why many regex was using this kind of double \ for escaping sequence,  this way is more convinient and seem to work as well

Reply:
It seems the same comment as in the other one : https://github.com/3liz/lizmap-web-client/pull/5434#discussion_r1963431020

@mdouchin FYI